acquia/mc-cs-plugin-custom-objects
# Maut 3912

See https://backlog.acquia.com/browse/MAUT-3912

#### Steps to test this PR:

Open some custom item detail page at https://gtest-beta.dev.mautic.net/s/custom/object/1/item to see if it is loading faster.

#### Other areas of Mautic that may be affected by the change:
1. Displaying/counting custom items in a contact detail page.
2. Displaying/counting custom items in a custom item detail page.
3. A linking search box in a contact detail page.
4. A linking search box in a custom item detail page.
5. Fulltext search of custom items.
